connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorHandle.java

```diff
@@ -54,6 +54,20 @@ public TaskHandle taskHandle(String taskId) {
         return taskHandles.computeIfAbsent(taskId, k -> new TaskHandle(this, taskId));
     }
 
+    /**
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!

Reply:
#6342 was merged. This PR is now rebased on top of it.